.github/workflows/build-rpm.yml

```diff
@@ -0,0 +1,65 @@
+name: Build SAF-CLI RPM Installers
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  build-deploy:
+    runs-on: ubuntu-22.04
+    container:
+      image: registry.access.redhat.com/ubi8/nodejs-18:latest
+      options: -u 0 # dnf needs root
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          path: saf
+
+      - name: setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version: "18"
+          check-latest: true
+          cache: 'npm'
+          cache-dependency-path: ./saf/package-lock.json
+
+      - name: install build dependencies
+        run: dnf install -y rpm-build perl-Digest-SHA # rpmbuild, shasum
+
+      - name: setup saf cli
+        working-directory: ./saf
+        run: |
+          rm -rf test
+          npm ci
+
+      - name: set filename envvar
+        working-directory: ./saf
+        run: |
+          echo "filename=saf-v$(cat VERSION)-linux-x64" >> "$GITHUB_ENV"
+
+      - name: pack into tarball # we are targetting x86-64 since oclif bundles a node binary, but the project is actually fully architecture independent so long as they have a node binary on it
+        working-directory: ./saf
+        run: npx oclif pack tarballs -t linux-x64 --no-xz
+
+      - name: repack tarball but include the spec file
```

Comment:
I assume if we have to do this that `oclif pack tarball` doesn't include the spec file?

Reply:
If we include it in the package.json's `files` attribute, then it's included in the tarball.  The problem is that I do not want to include the spec file in all of our releases.  

There were two solutions I could think of: 

1. Manual repacking (what I chose to do).
2. Edit the package.json file to include the spec file before calling the pack command.

I liked manually repacking since that meant I could a) avoid mucking around with checked-in files and b) so that I could set the filename for the tarball to be something consistent (i.e. not including the commit hash) when passed on to the rpm builder command.